Adds placeholder for Filecoin and Web3.Storage reference. [ciskip] (#847)
ipfs/ipfs-docs: docs/concepts/persistence.md

### Storing data with Filecoin

[Filecoin](https://filecoin.io) is a decentralized storage network in which storage providers rent their storage space to clients. The client and the storage provider agree on how much data will be stored, for how long, and at what cost. This agreement is called a _deal_. Once both parties agree to a deal, the client sends the data to the storage provider, who periodically verifies that they are correctly storing the data. When the client wants the data back, they send a request to the storage provider, who initiates the data transfer back to the client. For more information on how Filecoin works, head over to the [official Filecoin documentation →](https://docs.filecoin.io/about-filecoin/how-filecoin-works/)

> Filecoin provides users with a dependable, long-term storage solution. However, there are some limitations to consider. The retrieval process is not as fast as an IPFS pinning service, and the minimum file size accepted by a Filecoin storage provider can be several GiB. Also, the process for creating a storage deal may seem complicated to new users who aren't familiar with blockchain transactions or simply aren't comfortable working within a command line. If these are potential blockers for you, you may want to use [Web3.Storage](https://web3.storage).

Reviewer: I'd say "The retrieval process is not always as fast as an IPFS pinning service"

Reviewer 2: ```suggestion
Filecoin provides users with a dependable, long-term storage solution. However, there are some limitations to consider. The retrieval process is not always as fast as an IPFS pinning service, and the minimum file size accepted by a Filecoin storage provider can be several GiB. Also, the process for creating a storage deal may seem complicated to new users who aren't familiar with blockchain transactions or simply aren't comfortable working within a command line. If these are potential blockers for you, you may want to use [Web3.Storage](https://web3.storage).
```
@atopal Good idea. Resolved here.

> Filecoin provides users with a dependable, long-term storage solution. However, there are some limitations to consider. The retrieval process is not always as fast as an IPFS pinning service, and the minimum file size accepted by a Filecoin storage provider can be several GiB. Also, the process for creating a storage deal may seem complicated to new users who aren't familiar with blockchain transactions or simply aren't comfortable working within a command line. If these are potential blockers for you, you may want to use [Web3.Storage](https://web3.storage).

Reviewer 3: Instead of the last line I'd say: 

"Fortunately, there is a growing community of tools and service providers that help simplify the process of making content available over IPFS while persisting the data via FIlecoin. Below are a few options:

- Web3.Storage
- Estuary
- Powergate
- Chainsafe Storage
- Fleek

Reviewer 3: @jessicaschilling

Reviewer 2: Resolved in https://github.com/ipfs/ipfs-docs/pull/847/commits/539a1ceb2fe0ae879acb6857005f6fa3686c6e8c.

### Web3.Storage

[Web3.Storage](https://web3.storage) is a free service that makes it simple to store data using decentralized protocols by acting as both an IPFS pinning service and a Filecoin storage platform. When you upload a file to Web3.Storage, that file is immediately available for download. Periodically, the service will bundle data and create a deal with a reputable Filecoin storage provider, ensuring that the data is available in long-term storage.

> With Web3.Storage, users can quickly upload and download their files without worrying about file sizes or transaction costs. Files can be added using a simple web interface, and there's also an API to allow developers programmatic access to the service. All of these services are free to use. [Get started on Web3.Storage →](https://web3.storage)

Reviewer: This should emphasize that web3.storage is for developers first. Something like: 
"With Web3.Storage, developers can quickly upload and download their files without worrying about file sizes or costs using a simple JS client library, allowing developers programmatic access to Filecoin storage. Web3.storage also provides a simple web interface to manage and inspect files.

Reviewer 2: ```suggestion
With Web3.Storage, developers can quickly upload and retrieve files programmatically using a simple JavaScript client library, with no need to worry about file sizes or transaction costs. Web3.Storage also provides a simple web interface for managing and inspecting files. All of these services are free to use. [Get started on Web3.Storage →](https://web3.storage)
```
@atopal Good idea. Resolved here.

Reviewer 3: I'd kill this

- [Crust Network](https://crust.network/)

See how to [work with remote pinning services](../how-to/work-with-pinning-services.md).

## Long-term storage

> Storing data using a personal IPFS node is easy but inconvenient since you constantly have to manage your own hardware. This problem gave rise to pinning services that allow you to upload your data to a remotely-hosted IPFS node and retrieve it whenever you want. However, pinning services can be a dangerous form of long-term storage. Once a file, or collection of files, is no longer pinned, it may be lost entirely. There is no guarantee that the data is still accessible through IPFS. This is where [Filecoin](https://filecoin.io) comes in.

Reviewer 3: I think this may come across a little too derogatory. 

Maybe change to: 

However, while paying pinning providers can be convenient - it still requires an individual to bear the cost of storing that data (you paying a pinning provider). If that one sponsor stops paying for that pinning, the content may be lost entirely. While IPFS guarantees that any content on the network is discoverable, it doesn't guarantee that any content is persistently available. This is where [Filecoin](https://filecoin.io) comes in.

Author: That's fair.

```suggestion
However, while paying pinning providers can be convenient it still requires an individual to bear the cost of storing that data. If that one sponsor stops paying for the service, the content may be lost entirely. While IPFS guarantees that any content on the network is discoverable, it doesn't guarantee that any content is persistently available. This is where [Filecoin](https://filecoin.io) comes in.
```

How's this @jnthnvctr?

Author: Oh, nevermind. Just seen Jessica's comment below.

> Storing data using a personal IPFS node is easy but inconvenient since you constantly have to manage your own hardware. This problem gave rise to pinning services that allow you to upload your data to a remotely-hosted IPFS node and retrieve it whenever you want. However, pinning services can be a dangerous form of long-term storage. Once a file, or collection of files, is no longer pinned, it may be lost entirely. There is no guarantee that the data is still accessible through IPFS. This is where [Filecoin](https://filecoin.io) comes in.

Reviewer 2: ```suggestion
Storing data using a personal IPFS node is easy, but it can be inconvenient since you have to manage your own hardware. This problem gave rise to _pinning services_, paid services that allow you to upload your data to a remotely hosted IPFS node and retrieve it whenever you want. However, while paying a pinning service to store data is a convenient workaround, it still requires someone to bear the cost of storing that data. If that one sponsor stops paying for that pinning, the content may be lost entirely. While IPFS guarantees that any content on the network is discoverable, it doesn't guarantee that any content is persistently available. This is where [Filecoin](https://filecoin.io) comes in.
```
This addresses @jnthnvctr comments above.

Author: I prefer this suggestion. Thought's @jnthnvctr ?

### Storing data with Filecoin

> [Filecoin](https://filecoin.io) is a decentralized storage network where storage providers rent their storage space to clients. The client and the storage provider agree on how much data will be stored, how long, and the cost. This agreement is called a _deal_. Once both parties agree to the deal, the client sends the data to the storage provider, who periodically verifies that they are correctly storing the data. When the client wants the data back, they send a request to the storage provider who initiates the data transfer back to the client. For more information on how Filecoin works, head over to the [Filecoin documentation site →](https://docs.filecoin.io/about-filecoin/how-filecoin-works/)

Reviewer 2: ```suggestion
[Filecoin](https://filecoin.io) is a decentralized storage network in which storage providers rent their storage space to clients. The client and the storage provider agree on how much data will be stored, for how long, and at what cost. This agreement is called a _deal_. Once both parties agree to a deal, the client sends the data to the storage provider, who periodically verifies that they are correctly storing the data. When the client wants the data back, they send a request to the storage provider, who initiates the data transfer back to the client. For more information on how Filecoin works, head over to the [official Filecoin documentation →](https://docs.filecoin.io/about-filecoin/how-filecoin-works/)
```
grammar/clarity edits

> Filecoin provides users with a dependable long-term storage solution. However, there are some limitations. The retrieval process is not as fast as an IPFS pinning service, and the minimum file size can be several GiB. Also, the process for creating a storage deal can seem complicated to new users who aren't familiar with blockchain transactions or simply aren't comfortable working within a command line. To address these problems, Protocol Labs created Web3.Storage.

Reviewer 2: ```suggestion
Filecoin provides users with a dependable, long-term storage solution. However, there are some limitations to consider. The retrieval process is not as fast as an IPFS pinning service, and the minimum file size accepted by a Filecoin storage provider can be several GiB. Also, the process for creating a storage deal may seem complicated to new users who aren't familiar with blockchain transactions or simply aren't comfortable working within a command line. If these are potential blockers for you, you may want to use [Web3.Storage](https://web3.storage).
```
grammar, clarity, remove PL callout

> Filecoin provides users with a dependable long-term storage solution. However, there are some limitations. The retrieval process is not as fast as an IPFS pinning service, and the minimum file size can be several GiB. Also, the process for creating a storage deal can seem complicated to new users who aren't familiar with blockchain transactions or simply aren't comfortable working within a command line. To address these problems, Protocol Labs created Web3.Storage.

Reviewer 2: Resolved in https://github.com/ipfs/ipfs-docs/pull/847/commits/539a1ceb2fe0ae879acb6857005f6fa3686c6e8c.

### Web3.Storage

> [Web3.Storage](https://web3.storage) is a service that makes it incredibly simple to store data using decentralized protocols. It acts as both an IPFS pinning service and a Filecoin storage platform. When you upload a file to Web3.Storage, that file is immediately available for download. Periodically, the service will bundle data and create a deal with a Filecoin storage provider, ensuring that the data is available in long-term storage.

Reviewer 2: ```suggestion
[Web3.Storage](https://web3.storage) is a free service that makes it simple to store data using decentralized protocols by acting as both an IPFS pinning service and a Filecoin storage platform. When you upload a file to Web3.Storage, that file is immediately available for download. Periodically, the service will bundle data and create a deal with a reputable Filecoin storage provider, ensuring that the data is available in long-term storage.
```
grammar, clarity, less marketing-speak, emphasize free

> [Web3.Storage](https://web3.storage) is a service that makes it incredibly simple to store data using decentralized protocols. It acts as both an IPFS pinning service and a Filecoin storage platform. When you upload a file to Web3.Storage, that file is immediately available for download. Periodically, the service will bundle data and create a deal with a Filecoin storage provider, ensuring that the data is available in long-term storage.

Reviewer 2: Resolved in https://github.com/ipfs/ipfs-docs/pull/847/commits/539a1ceb2fe0ae879acb6857005f6fa3686c6e8c.

> Using Web3.Storage users can quickly upload and download their files without worrying about files sizes or transaction costs. There's even an API to allow developers programmatic access to the service. To use Web3.Storage, [head over to Web3.Storage and sign up for free →](https://web3.storage)

Reviewer 2: ```suggestion
With Web3.Storage, users can quickly upload and download their files without worrying about file sizes or transaction costs. Files can be added using a simple web interface, and there's also an API to allow developers programmatic access to the service. All of these services are free to use. [Get started on Web3.Storage →](https://web3.storage)
```
grammar, clarity, emphasize free
